webpack.config.js

```diff
@@ -78,16 +77,10 @@ const webpackConfig = {
 		chunkFilename: `chunks/[name]${ suffix }.js`,
 		path: path.join( __dirname, 'dist' ),
 		library: [ 'wc', '[modulename]' ],
-		libraryTarget: 'this',
-		jsonpFunction: '__wcAdmin_webpackJsonp',
+		libraryTarget: 'window',
```

Comment:
This libraryTarget is used in dependency extraction, I don't know why it was `this` before, but it being window is needed to properly attach exports to window.